Separate documentation for WebGL 1 and 2 methods/properties (#9699)
mdn/content: files/en-us/web/api/webglrenderingcontext/bufferdata/index.md

---
{{APIRef("WebGL")}}

> > **Note:** This page describes the WebGL 1 version of this method.  Click [here](/en-US/docs/Web/API/WebGL2RenderingContext/bufferData) to read about the WebGL 2 version.

Reviewer: ```suggestion
> **Note:** This page describes the WebGL 1 version of this method. [`WebGL2RenderingContext.bufferData()`](/en-US/docs/Web/API/WebGL2RenderingContext/bufferData) is the WebGL 2 version of this method.
```
